client/signup/index.web.js

```diff
@@ -58,11 +58,21 @@ export default function() {
 	page( '/jetpack/connect/store', jetpackConnectController.plansLanding );
 	page( '/jetpack/connect/store/:intervalType', jetpackConnectController.plansLanding );
 
-	page( '/jetpack/connect/vaultpress', jetpackConnectController.vaultpressLanding );
-	page( '/jetpack/connect/vaultpress/:intervalType', jetpackConnectController.vaultpressLanding );
-
-	page( '/jetpack/connect/akismet', jetpackConnectController.akismetLanding );
-	page( '/jetpack/connect/akismet/:intervalType', jetpackConnectController.akismetLanding );
+	page( '/jetpack/connect/vaultpress', '/jetpack/connect/store' );
+	page( '/jetpack/connect/vaultpress/:intervalType', context => {
```

Comment:
This arrow function is the same for both the akismet and vaultpress redirect. Care to define it only once instead?